hardhat.config.ts

```diff
@@ -33,6 +35,8 @@ function getChainConfig(chain: keyof typeof chainKeyToSlug): NetworkUserConfig {
   return {
     accounts: [`0x${privateKey}`],
     chainId: chainKeyToSlug[chain],
+    gasPrice: gasPrice[chain] ? gasPrice[chain] : "auto",
+    gasMultiplier: gasMultiplier[chain] ? gasMultiplier[chain] : 1,
```

Comment:
We make providers in the script and dont use hardhat network. So these overrides will not work.